### DISCUSSION TITLE
Navigation quick options for nearby favorites, fuel stations, speed cameras and elevation diagram

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#416] [Q&A] [answered]
We can improve the navigation options (_long press direction_) to turn on / off:
- Nearby favorites
- Nearby points of interest

So we can see them on demand while traveling (and simplify app settings).

<img src="https://github.com/devemux86/cruiser/assets/3484020/ddd320b9-eecd-4b7e-9fc7-5c4511c5a99e" width="50%"/>

**Comment by WolfgangZ42** [comment]
Great - now you can easily switch on the gas station display only when it's really necessary. Superb idea!

**Comment by devemux86** [comment]
We could probably split the options per favorite group (that participates in navigation) or POI type,
but let's start with a simple implementation to see how good it works. :slightly_smiling_face:

**Comment by MotoUKRider** [comment]
This one confused me with beta-6 just now - I couldn't turn it off because the slider wouldn't go down to "0".  It was only looking for this thread that made me realize how to disable "Nearby Favorite/POI" on the panels during Navigation.

The jist of my post is that it *might* be confusing that you can set a numeric value but not "off" in the settings - relying on a long-press to find the action to enable/disable.

**Reply by devemux86** [reply]
@MotoUKRider 

Navigation options are more about doing it quickly while we're on the move.

The main way to turn on / off the nearby points is on their screens:
- Nearby favorites: in their groups separately per group
- Nearby POI: in navigation display settings per POI type

**Comment by MotoUKRider** [comment]
With the addition of "Route diagram" on this dialog, there's now a spare panel available - could the Waypoint List be added? (minor issue is that the pin icon clashes with existing POI panel).

It would be nice to bring the route itinerary up in less than 4 taps :crossed_fingers:

**Reply by devemux86** [reply]
@MotoUKRider do you mean the waypoints list or the turn instructions list?

Also note that this is the navigation _options_ screen, displaying on / off switches for navigation options.

**Reply by MotoUKRider** [reply]
@devemux86 Yeah, I was hoping that you wouldn't notice that :smile:   Perhaps on the *Short Tap* dialog, but then we need another panel to make the layout right.  Meh!

Waypoints List, ie: list of Shaping-Points+Waypoints.

**Comment by devemux86** [comment, marked as answer]
Navigation quick options for nearby favorites and points of interest is available in [Cruiser 3.0.38](https://play.google.com/store/apps/details?id=gr.talent.cruiser).

(long press direction icon)
